.platform/solr_config/stopwords_en.txt

```diff
@@ -0,0 +1,35 @@
+a
+an
+and
+are
+as
+at
+be
+but
+by
+for
+if
+in
+into
+is
+it
+no
+not
+of
+on
```

Comment:
TODO: compare with D7 solr config stopwords.  Defo need to remove 'on' so that searches for "go on ni" work.